iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Frames
+import PassKit
```

Comment:
Not part of this PR

Reply:
@ehab-al-cko , please review your PR

Reply:
@alex-ioja-yang-cko I thought adding the PassKit integration in sample app makes sense and improves our sample app. Why do you think we should not do this now ?